user/caching.md

```diff
@@ -287,11 +288,11 @@ jobs should use.
 These factors are:
 
 1. OS name (currently, `linux` or `osx`)
-1. OS distribution (for Linux, `precise` or `trusty`)
-1. OS X image name (e.g., `xcode7.2`)
-1. Names and values of visible environment variables set in `.travis.yml` or Settings panel
-1. Language runtime version (for the language specified in the `language` key) if applicable
-1. For Bundler-aware jobs, the name of the `Gemfile` used
+2. OS distribution (for Linux, `precise` or `trusty`)
```

Comment:
I think it is nicer. You can of course add a `1.` in the middle and it will still work, but will look a little odder.